storage/database/entity/bill.go

```diff
@@ -0,0 +1,32 @@
+package entity
+
+import (
+	"github.com/google/uuid"
+	. "split-the-bill-server/domain/model"
+	"time"
+)
+
+type Bill struct {
+	Base
+	Name    string `gorm:"not null"`
```

Comment:
Don't you think we should store the ownerId as well?

Reply:
Yes good idea.